packages/account-abstraction/src/BiconomyPaymasterAPI.ts

```diff
@@ -1,22 +1,15 @@
 import { resolveProperties } from '@ethersproject/properties'
 import { UserOperation } from '@biconomy-sdk/core-types'
-import { hexConcat } from 'ethers/lib/utils'
 import { HttpMethod, sendRequest } from './utils/httpRequests'
-export class PaymasterAPI {
-  // Might maintain API key at smart account level
-  constructor(
-    readonly apiUrl: string,
-    readonly dappAPIKey: string,
-    readonly payMasterAddress: string
-  ) {
+import { IPaymasterAPI } from '@biconomy-sdk/core-types'
+
+export class BiconomyPaymasterAPI implements IPaymasterAPI {
+  constructor(readonly apiUrl: string, readonly dappAPIKey: string) {
```

Comment:
apiUrl should be renamed to signingServiceUrl

Reply:
done